packages/data-client-core/vite.config.ts

```diff
@@ -0,0 +1,3 @@
+import { viteConfig as defineBaseViteConfig } from '@terra-ui-packages/build-utils';
```

Comment:
The build errors are because these imports need to be updated.

```suggestion
import { defineBaseViteConfig } from '@terra-ui-packages/build-utils';
```